pkg/virt-launcher/virtwrap/converter/network.go

```diff
@@ -151,8 +151,8 @@ func CreateDomainInterfaces(vmi *v1.VirtualMachineInstance, domain *api.Domain,
 
 func GetInterfaceType(iface *v1.Interface) string {
 	if iface.Slirp != nil {
-		// Slirp configuration works only with e1000 or rtl8139
-		if iface.Model != "e1000" && iface.Model != "rtl8139" {
+		// Slirp configuration works with e1000 only
+		if iface.Model != "e1000" {
```

Comment:
I do not understand this change.
Based on what `rt18139` is now overridden by `e1000`?

Reply:
Sorry, the correct changes were probably overridden by the rebase, the correct fix is in place now.